pages/blog/hacktoberfest-summary-2020.md

```diff
@@ -0,0 +1,430 @@
+
+Start small. Don't start with tasks that can be overwhelming. Don't throw yourself into the deep end. 
+
+We wanted to help others make first baby steps in a secure and inclusive environment, with lots of patience and support.
+
+## What It Takes To Have 70 PRs Merged
+
+It is not enough to label 70 issues with the **hacktoberfest** label, sorry :smiley:
+
+First of all, you need to be passionate about open source and dedicated to what you do. It can't just be a task that somebody assigned you. It would help if you were prepared to treat Hacktoberfest participants as a priority. I would compare it to the onboarding process of new hires. 
+
+Of course, not all participants join to stay longer, usually it is just one pull request, and I'm out of here.
+
+It doesn't matter. 
+
+Please don't make assumptions; assumptions are evil. Be opened, treat every contribution equally, and remember that the onboarding process is a crucial element. If you fail with the onboarding process, you fail big time at the very beginning.
+
+### What We Prepared
+
+We prepared the following materials for potential participants:
+- [Blog post](https://www.asyncapi.com/blog/hacktoberfest-2020) about our participation,
+- [Onboarding videos](https://www.youtube.com/playlist?list=PLbi1gRlP7pigFSE_6G88x0t12HFLU4A4b) that explain how to start,
+- 78 issues from more than 30 repositories and put them all in one [list](https://docs.google.com/spreadsheets/d/1vX4J395apexutfQ0OSqPNltFKDacmemHZwCmOXwHNLo/) with additional information about difficulty level or technical area it is about, 
+
+It took me around eight workdays to do it all. The most time consuming was to go through all the issues, pick candidates, create new ones, and group them all in a google sheet.
+
+### What We Did During The Event
+
+70 pull requests mean 70 reviews :sweat:
+
+If you know your project well, it is not very time consuming, though, and anyway, it is work you have to do as a maintainer. I do not count this time as an extra Hacktoberfest effort. Of course it can be overwhelming if this is not a standard amount of PRs that you get every month.
+
+We also hosted [office hours](https://www.youtube.com/playlist?list=PLbi1gRlP7pigZP2da6zbDT2YU7glQOTPH) for participants. This was fun, and we anyway wanted to start doing live streams about AsyncAPI.
+
+Last but not least, once a week, I advertised our project on the official Hacktoberfest Discord server.
+
+[![](/img/posts/hacktoberfest-summary-2020/feedback-sharkham.webp)](https://github.com/asyncapi/vscode-extension/pull/8#issuecomment-719759737)
+
+It looks like we do not need to do more for the next year.
+
+## Was It Worth It
+
+Hell yeah, and I'm already looking forward to Hacktoberfest 2021.
+
+It was great to see so many different people engaging with the project and seeing we reached our goal.
```

Comment:
are you sure `involved the project` is fine?

Reply:
I think `involved` fits since they were involved in project activities during Hacktoberfest, BUT I might have gone a bit too far with this one :) maybe `interacting with` or `participating in`?

Reply:
`interacting` sounds great